File: backend/config/urls.py
from django.conf import settings
from django.conf.urls.static import static
from django.urls import include, path

from rest_framework.permissions import AllowAny

from drf_yasg import openapi
from drf_yasg.views import get_schema_view

schema_view = get_schema_view(openapi.Info(title='AutoParkApi', default_version='v1', description='About AutoParks',
                                           contact=openapi.Contact(email='admin@gmail.com')), public=True,
                              permission_classes=[AllowAny])
urlpatterns = [
    path('api/auth', include('apps.auth.urls')),
    path('api/users', include('apps.users.urls')),
    path('api/meters', include('apps.meter.urls')),
    path('api/readings', include('apps.readings.urls')),
    path('api/advertisement', include('apps.advertisement.urls')),
    path('api/poll', include('apps.poll.urls')),
    path('api/news', include('apps.news.urls')),
    path('api/fault_msg', include('apps.fault_message.urls')),
    path('api/payment', include('apps.payment_for_user.urls')),
    path('api/doc', schema_view.with_ui('swagger', cache_timeout=0))
]
urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)

File: frontend/src/configs/urls.js
const baseURL = 'api';

const auth = '/auth';
const users = '/users';
const readings = '/readings';
const advertisement = '/advertisement';
const poll = '/poll';
const meters = '/meters';
const news = '/news';
const fault_msg = '/fault_msg';
const payment = '/payment';

const mainUrls = {
    auth: {
        login: auth,
        register: `${auth}/register`,
        refresh: `${auth}/refresh`,
        me: `${auth}/me`,
        activate: `${auth}/activate`
    },
    users: {
        users: users,
        profile: `${users}/profile`,
    },

    readings: {
        readings,
    },
    advertisement: {
        advertisement,
        byId: (id) => `${advertisement}/${id}`,
        addPhoto: (id) => `${advertisement}/${id}`,

    },
    poll: {
        poll,
        byId: (id) => `${poll}/${id}`,
    },
    meters: {
        meters,
    },

    news: {
        news,
        byId: (id) => `${news}/${id}`,
    },

    fault_msg: {
        fault_msg,
    },

    payment: {
        payment,
    },

}

export {baseURL, mainUrls};